tests/test_search_pattern.py

```diff
@@ -0,0 +1,172 @@
+"""
+Tests the SearchPattern class
+"""
+
+import pytest
+
+from modules import decision_command
+from modules import odometry_and_time
+from modules import drone_odometry_local
+from modules.decision import search_pattern
+
+
+@pytest.fixture()
+def search_pattern_width_greater_height():
+    """
+    Initializes a SearchPattern instance.
+    """
+    search_pattern_instance = search_pattern.SearchPattern(
+        camera_fov_forwards=30.0,
+        camera_fov_sideways=60.0,
+        search_height=100.0,
+        search_overlap=0.2,
+        current_position_x=100.0,
+        current_position_y=50.0,
+        distance_squared_threshold=2.0,
+        small_adjustment=1,
```

Comment:
if you use these numbers to initialize every instance of the class, use a constant